cmd/machine/status.go

```diff
@@ -61,7 +61,7 @@ func (cmd *StatusCmd) Run(ctx context.Context, args []string) error {
 		} else if machineStatus == client.StatusBusy {
 			log.Default.Infof("Machine '%s' is '%s', which means its currently unaccessible. This is usually resolved by waiting a couple of minutes", machineClient.Machine(), machineStatus)
 		} else if machineStatus == client.StatusNotFound {
-			log.Default.Infof("Machine '%s' is '%s'", machineClient.Machine(), machineStatus)
+			log.Default.Infof("Machine '%s' is '%s', which means it can't find.", machineClient.Machine(), machineStatus)
```

Comment:
could you change that back please? `StatusNotFound` already indicates that it can't be found imo

Reply:
Of course.

Reverted. Please review again.